INN/umbrella-sfpublicpress
# Remove job title from article bylines, fix email button in safari

**Should be reviewed after #92 is merged because it's branched off of that branch**

## Changes

This pull request makes the following changes:

- Removes the job title from bylines on single posts
- Fixes the bug in safari that was cutting off the email icon on single post bylines

![Screen Shot 2020-06-23 at 1 34 28 PM](https://user-images.githubusercontent.com/18353636/85436006-58367400-b556-11ea-9588-da234337d5b1.png)

## Why



For #94 and #93 

## Testing/Questions

Features that this PR affects:

- Single post bylines


Questions that need to be answered before merging:


Steps to test this PR:

1. View a single post and make sure the authors job title isn't displayed
2. View a single post on safari and make sure the email button looks ok
